spec/features/admin/admin_can_manually_create_playlist_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
```

Comment:
This test looks good - It looks like you caught everything, and I can't think of anything else to add here